src/main.go

```diff
@@ -29,26 +28,31 @@ import (
 	"github.com/zairza-cetb/bench-routes/src/metrics/journal"
 	"github.com/zairza-cetb/bench-routes/src/metrics/process"
 	sysMetrics "github.com/zairza-cetb/bench-routes/src/metrics/system"
+	"github.com/zairza-cetb/bench-routes/tsdb"
+	"github.com/zairza-cetb/bench-routes/tsdb/v1"
 )
 
 var (
 	port                        = ":9990" // default listen and serve at 9090
 	enableProcessCollection     = false   // default collection of process metrics in host of bench-routes
 	processCollectionScrapeTime = time.Second * 5
 	defaultScrapeTime           = time.Second * 3
+	storageFile                 = "000000"
 	systemMetricsPath           = "storage/system.json"
 	journalMetricsPath          = "storage/journal.json"
+	filesPath                   = "storage/" + storageFile
```

Comment:
nit - You can use the variable `storageDir` defined on line 63

Reply:
@chetanyakan this will be rewritten by binary tsdb PR, so I think that's why we arent updating this PR. It awesome to receive PR reviews. Thank you for that.